ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -607,7 +619,7 @@
 
 (defn- max-date-within-90-days? [{max-date :max-date}]
   (and max-date
-       (.isBefore (.toLocalDate max-date) (.plusDays (java.time.LocalDate/now) 90))
+       (.isBefore (.toLocalDate max-date) (.plusDays (java.time.LocalDate/now) 180))
```

Comment:
Function name will be misleading after this, I suggest renaming it since it's referenced in only one place in the code. E.g.
```
(defn- max-date-within-evaluation-window? [{max-date :max-date}]
```

Reply:
Good idea!